public_html/wp-content/plugins/wc-post-types/package.json

```diff
@@ -22,13 +22,14 @@
 		"lint:js": "wp-scripts lint-js js/src",
 		"lint:pkg-json": "wp-scripts lint-pkg-json"
 	},
+	"eslintConfig:comment": {
+		"root": "Setting this to true is necessary to avoid merging WordCamp.org configs with any that exist in parent folders in the developer's environment. See https://github.com/eslint/eslint/commit/c99e9b621fe46b8535cf59fe489977dd36cb2a39."
+	},
 	"eslintConfig": {
-		"extends": "plugin:@wordpress/eslint-plugin/recommended",
+		"root": true,
+		"extends": "../../mu-plugins/blocks/.eslintrc.js",
```

Comment:
This is unrelated to the PR — can you take this out before merging?

If we want to share one customized eslint config across the whole project, we should make that the project root, not pull in from a separate project like this.

Reply:
I don't see any reason for us to have inconsistent standards within the same codebase. I agree that it'd be best to have 1 file at the root, but that would be ever farther out of scope. Doing this much was necessary for this PR since it directly affected the new code this PR adds.

This moves us closer to having 1 standard, and is very easy to implement. We can always iterate later to move to a single file.

Since the block is time-sensitive, I'll go ahead and merge the other commits, and leave this one just in this branch, so that we can continue discussing delaying the approved commits.

Reply:
I agree we should stick to one eslint config, and if you push this commit to another PR we can move forward.

> Since the block is time-sensitive, I'll go ahead and merge the other commits, and leave this one just in this branch, so that we can continue discussing delaying the approved commits.

That's exactly what I was suggesting.

Reply:
Well, I see after I commented that … this isn't _exactly_ what I meant. Now that you've merged those commits, this is full of conflicts. Now you can close this branch, create a new branch off production, cherry pick this commit, and open a new PR to make this change 👍

Reply:
BTW, i'm happy to kick that off if you're busy with more urgent things.

Reply:
That would be great, thank you!

Reply:
Rebase resolved the conflicts.